Traceback (most recent call last):
  File "/home/runner/.cache/pypoetry/virtualenvs/medusa-2VgPi4zZ-py3.10/lib/python3.10/site-packages/jupyter_cache/executors/utils.py", line 58, in single_nb_execution
    executenb(
  File "/home/runner/.cache/pypoetry/virtualenvs/medusa-2VgPi4zZ-py3.10/lib/python3.10/site-packages/nbclient/client.py", line 1305, in execute
    return NotebookClient(nb=nb, resources=resources, km=km, **kwargs).execute()
  File "/home/runner/.cache/pypoetry/virtualenvs/medusa-2VgPi4zZ-py3.10/lib/python3.10/site-packages/jupyter_core/utils/__init__.py", line 166, in wrapped
    return loop.run_until_complete(inner)
  File "/usr/lib/python3.10/asyncio/base_events.py", line 646, in run_until_complete
    return future.result()
  File "/home/runner/.cache/pypoetry/virtualenvs/medusa-2VgPi4zZ-py3.10/lib/python3.10/site-packages/nbclient/client.py", line 705, in async_execute
    await self.async_execute_cell(
  File "/home/runner/.cache/pypoetry/virtualenvs/medusa-2VgPi4zZ-py3.10/lib/python3.10/site-packages/nbclient/client.py", line 1058, in async_execute_cell
    await self._check_raise_for_error(cell, cell_index, exec_reply)
  File "/home/runner/.cache/pypoetry/virtualenvs/medusa-2VgPi4zZ-py3.10/lib/python3.10/site-packages/nbclient/client.py", line 914, in _check_raise_for_error
    raise CellExecutionError.from_cell_and_msg(cell, exec_reply_content)
nbclient.exceptions.CellExecutionError: An error occurred while executing the following cell:
------------------
from medusa.data import get_example_video
from IPython.display import Video  # just to show the video in the notebook/website

vid = get_example_video()
f_out = './smooth.mp4'

renderer(f_out, data_4d, video=vid)

# Show result
Video(f_out, embed=True)
------------------


[0;31m---------------------------------------------------------------------------[0m
[0;31mTypeError[0m                                 Traceback (most recent call last)
Cell [0;32mIn[4], line 7[0m
[1;32m      4[0m vid [38;5;241m=[39m get_example_video()
[1;32m      5[0m f_out [38;5;241m=[39m [38;5;124m'[39m[38;5;124m./smooth.mp4[39m[38;5;124m'[39m
[0;32m----> 7[0m [43mrenderer[49m[43m([49m[43mf_out[49m[43m,[49m[43m [49m[43mdata_4d[49m[43m,[49m[43m [49m[43mvideo[49m[38;5;241;43m=[39;49m[43mvid[49m[43m)[49m
[1;32m      9[0m [38;5;66;03m# Show result[39;00m
[1;32m     10[0m Video(f_out, embed[38;5;241m=[39m[38;5;28;01mTrue[39;00m)

[0;31mTypeError[0m: 'VideoRenderer' object is not callable

